package.json

```diff
@@ -24,13 +24,14 @@
     "@atlaskit/icon": "7.0.0",
     "@atlaskit/inline-dialog": "3.2.0",
     "@atlaskit/modal-dialog": "2.1.2",
-    "@atlaskit/multi-select": "6.0.0",
+    "@atlaskit/multi-select": "6.2.0",
     "@atlaskit/spinner": "2.2.3",
     "@atlaskit/tabs": "2.0.0",
     "@atlassian/aui": "6.0.6",
     "async": "0.9.0",
     "autosize": "1.18.13",
     "bootstrap": "3.1.1",
+    "debounce": "1.0.2",
```

Comment:
@virtuacoplenny omg, didn't notice we already depend on lodash, I wanted to use exactly the lodash one, but then thought I'm not going to add such a bug dependency just for debounce. I'll make the change. Thanks for pointing this out!